Libraries/Storage/AsyncStorage.js

```diff
@@ -286,6 +325,32 @@ var AsyncStorage = {
    * json. Returns a `Promise` object.
    *
    * Not supported by all native implementations.
+   *
+   * Example:
+   * ```JavaScript
+   *   let key_1 = 'k1'
+   *   let val_1a = JSON.stringify({'a': '123', 'b': {'b1': '234', 'b2': '234'}})
+   *   let val_1b = JSON.stringify({'b': {'b2': '555', 'b3': '555'}})
+   *
+   *   let key_2 = 'k2'
+   *   let val_2a = val_1a
+   *   let val_2b = val_1b
+   *
+   *   let multi_set_keys    = [[key_1, val_1a],[key_2, val_2a]]
+   *   let multi_merge_pairs = [[key_1, val_1b],[key_2, val_2b]]
+   *
+   *   AsyncStorage.multiSet(multi_set_keys, (err) => {
+   *     console.log('[multiSet] done!')
+   *     AsyncStorage.multiMerge(multi_merge_pairs, (err) => {
+   *       console.log('[multiMerge] done!')
+   *       AsyncStorage.multiGet([key_1,key_2], (err, stores) => {
+   *         stores.map( (result, i, store) => {
+   *           console.log('store["%s"] contains...', store[i][0], store[i][1])
+   *         })
+   *       })
+   *     })
+   *   })
```

Comment:
Maybe I'm missing something but this example doesn't make it clearer to me what multimerge does.